components/function-controller/pkg/apis/serverless/v1alpha2/function_types.go

```diff
@@ -232,47 +236,47 @@ const (
 )
 
 type Condition struct {
-	// Type of function condition.
+	// Specifies the type of the Function condition.
```

Comment:
```suggestion
	// Specifies the type of the Function's condition.
```
You were right - it sounds better :)